email-log.php

```diff
@@ -1,11 +1,11 @@
 <?php
 /**
  * Plugin Name: Email Log
- * Plugin URI: http://sudarmuthu.com/wordpress/email-log
+ * Plugin URI: https://wpemaillog.com
  * Description: Logs every email sent through WordPress
  * Donate Link: http://sudarmuthu.com/if-you-wanna-thank-me
  * Author: Sudar
- * Version: 1.9.1
```

Comment:
Let's not make the version change as part of this commit. I will change the version here and in other places where I make the release. It is part of my build process.